<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TitleEntryDialog</class>
 <widget class="QDialog" name="TitleEntryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>152</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>327</width>
    <height>152</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>327</width>
    <height>152</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Title Entry</string>
  </property>
  <widget class="QLabel" name="boxArtImg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>16</y>
     <width>85</width>
     <height>120</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Downloading image...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>211</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="nameLbl">
      <property name="text">
       <string>&lt;b&gt;Name: &lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="titleIDLbl">
      <property name="text">
       <string>&lt;b&gt;TitleID: &lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="gamerscoreLbl">
      <property name="text">
       <string>&lt;b&gt;Gamerscore: &lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="achievementsLbl">
      <property name="text">
       <string>&lt;b&gt;Achievements: &lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="avatarAwardsLbl">
      <property name="text">
       <string>&lt;b&gt;Avatar Awards: &lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="flagsLbl">
      <property name="text">
       <string>&lt;b&gt;Flags: &lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lastPlayedLbl">
      <property name="text">
       <string>&lt;b&gt;Last Played: &lt;/b&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
